StereoKitC/ui/stereokit_ui.cpp

```diff
@@ -227,9 +227,13 @@ void ui_image(sprite_t image, vec2 size) {
 }
 
 ///////////////////////////////////////////
-
 template<typename C>
 void _ui_button_img_surface(const C* text, sprite_t image, ui_btn_layout_ image_layout, text_align_ text_layout, vec3 window_relative_pos, vec2 size, float finger_offset) {
+	_ui_button_img_surface(text, image, image_layout, text_layout, window_relative_pos, size, finger_offset, skui_tint);
+}
```

Comment:
I think it might be better to just remove this `_ui_button_img_surface` overload and update the rest of the code to use the new one!

I've also been thinking it might be nice to drop `skui_tint` from button image sprites entirely. I tried this in practice recently, and it felt odd. Let's just use `color128{1,1,1,1}` for the default value here!